website/docs/mutation.md

```diff
@@ -0,0 +1,281 @@
+---
+id: mutation
+title: Mutation
+status: alpha
```

Comment:
I don't think `status` is supported as a header item https://v2.docusaurus.io/docs/markdown-features#markdown-headers

can we move this down?

Reply:
Moved down to after the first paragraph. Although it's now a bit redundant with what we have in the text.